src/hotspot/share/gc/shared/stringdedup/stringDedupProcessor.cpp

```diff
@@ -187,6 +196,10 @@ void StringDedup::Processor::run(JavaThread* thread) {
     cleanup_table(false /* grow_only */, StringDeduplicationResizeALot /* force */);
     _cur_stat.report_active_end();
     log_statistics();
+    if (UsePerfData && os::is_thread_cpu_time_supported()) {
+      ThreadTotalCPUTimeClosure tttc(_concurrent_dedup_thread_cpu_time, true);
```

Comment:
I think it is better to not classify StringDedup thread as a GC thread, so remove the "true" parameter.
Although StringDedup requests are initiated during GC (search `_string_dedup_requests.add`), they are not part of the GC.